CHANGELOG.md

```diff
@@ -14,18 +16,19 @@
 * File format: Generates Realms with file format v23.
 * Realm Studio 13.0.0 or above is required to open Realms created by this version.
 * This release is compatible with the following Kotlin releases:
-  * Kotlin 1.7.20 and above.
+  * Kotlin 1.8.0 and above.
```

Comment:
Should we spell out that our plugin is not FIR compatible yet ... 🤔

Reply:
Yes, probably a good idea